rts/lra/jaxrs/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -554,6 +550,15 @@ public void filter(ContainerRequestContext requestContext, ContainerResponseCont
         }
     }
 
+    private NarayanaLRAClient getLRAClient() {
+        if (lraClient == null) {
+            // no need to lock
+            lraClient = new NarayanaLRAClient();
```

Comment:
my 2pc: while I think this is not fully safe as it could still potentially provide a partially initialized object it is probably not be a trouble in later usages

Reply:
Java guarantees that assigning a constructed object to a variable will always result in the variable referencing a fully initialised Java object.